tests/test_utils/test_docker_entrypoint.py

```
@@ -0,0 +1,98 @@
+    deadline = time.monotonic() + timeout
+    while time.monotonic() < deadline:
+        if path.exists():
+            return True
+        time.sleep(0.01)
+    return False
+
+
+def test_entrypoint_gives_the_command_a_real_stdin(tmp_path: Path) -> None:
+    env = _fake_modelconverter(tmp_path, 'read -r line\nexit "$line"\n')
+
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)